clove/network/bitcoin_based/ravencoin.py

```diff
@@ -23,12 +22,8 @@ class Ravencoin(InsightAPIv4, BitcoinBaseNetwork):
         'SECRET_KEY': 128
     }
     source_code_url = 'https://github.com/RavenProject/Ravencoin/blob/master/src/chainparams.cpp'
-    api_url = 'https://ravencoin.network'
-    api_prefix = '/api'
-
-    @classmethod
-    def get_fee(cls):
-        return clove_req_json(f'{cls.api_url}{cls.api_prefix}/utils/estimatesmartfee?nbBlocks=1')['1']
```

Comment:
Why is this removed?

Reply:
This method was here to support the case when there was a different prefix for API and UI. I created two variables and now we have `api_url`, `ui_url` and `InsightAPIv4` is doing the rest.

Reply:
You are right, Monacoin has a different endpoint for fee. I'll revert this change.